main.tf

```diff
@@ -80,7 +80,10 @@ resource "aws_iam_policy" "idp_guardrail" {
       {
         "Sid" : "DenyPermBoundaryIAMIDPAlteration",
```

Comment:
Let's make the naming more generic now that is not just for IPDs. e.g. `DenyPermBoundaryBCGOVIAMAlteration`

Same thing with tf resource `resource "aws_iam_policy" "idp_guardrail"` -> `resource "aws_iam_policy" "bcgov_perm_boundary"`